Validation/HGCalValidation/python/HGCalValidator_cfi.py

```diff
@@ -0,0 +1,46 @@
+import FWCore.ParameterSet.Config as cms
+
+from Validation.HGCalValidation.CaloParticleSelectionForEfficiency_cfi import *
+from Validation.HGCalValidation.HGVHistoProducerAlgoBlock_cfi import *
+
+hgcalValidator = cms.EDAnalyzer(
+    "HGCalValidator",
+
+    ### general settings ###
+    # selection of CP for evaluation of efficiency #
+    CaloParticleSelectionForEfficiency,
+
+    ### reco input configuration ###
+    #2dlayerclusters, pfclusters, multiclusters
+    #label = cms.VInputTag(cms.InputTag("hgcalLayerClusters"), cms.InputTag("particleFlowClusterHGCal"), cms.InputTag("hgcalMultiClusters") ),
+    label = cms.VInputTag(cms.InputTag("hgcalLayerClusters")),
+    
+    #CaloParticle related plots
+    doCaloParticlePlots = cms.untracked.bool(True),
+    dolayerclustersPlots = cms.untracked.bool(True),
+
+    #The cumulative material budget in front of each layer. To be more specific, it 
+    #is the material budget just in front of the active material (not including it). 
+    #This file is created using the official material budget code. 
+    cummatbudinxo = cms.FileInPath('Validation/HGCalValidation/data/D28.cumulative.xo'),
```

Comment:
it's going to be difficult to keep this manually in sync with all the different 2023 geometry versions...

Reply:
Yes, you are right, but could we please keep this? Upon request and on specific geometry versions I could produce the 
relevant file. It is that in previous beam tests they have used the longitudinal depth barycentre (which uses this file), while it 
would be nice to connect the material budget package with the validation package.

Reply:
Will this code be run by default for any phase 2 workflow? What will happen if this is run on a geometry with a different material budget?

Reply:
The longitudinal depth barycentre plot will be wrong. This one [here](http://apsallid.web.cern.ch/apsallid/HGCalValidationPlots/plots_PDGId22_nPart1_Pt25Eta1p6to2p8_Miscellaneous/longdepthbarycentre_longdepthbarycentre_zminus.png). 

This is the only usage of the file currently, that is to produce that plot. Specifically, a std::map<double, double> cummatbudg -> (layer, accumulatedmaterialbudgetuptothatlayer)
is filled by that file. Then, at the core of the code, layer number is accessed as usual like: 

layer = recHitTools_->getLayerWithOffset(rh_detid);

So, since this is a map even if the key doesn't exist, it shouldn't crash, right?

Reply:
I'm more worried about getting wrong/nonsensical results out of the validation code. If this is acceptable to the people doing HGCal validation, we can continue in this way, but in practice it will probably mean the validation code is frequently out of sync with the latest version of the geometry.

Reply:
Ok, if you think it would bring confusion I can remove that plot, the file, the data directory that was created for that reason and any relevant code. In other case, I can add to the title of the plot I sent you before something like "always check that you read the correct material budget file". What do you think?

Reply:
For me, a title message like that would be fine. I would specify in the title message which material budget file was used, so an observer does not have to dig through to find the python config.